packages/react-core/src/Authenticator/hooks/useAuthenticator/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const USE_AUTHENTICATOR_ERROR =
+  '`useAuthenticator` must be used inside an `Authenticator.Provider`.';
```

Comment:
Does this need to be added to our i18n translations?

Reply:
We don't translate console warning messages